pom.xml

```diff
@@ -814,7 +812,17 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <configuration>
             <forkCount>${surefire.forkCount}</forkCount>
+            <reuseForks>true</reuseForks>
             <excludedGroups>${surefire.excludedGroups}</excludedGroups>
+            <!-- Tests to exclude due to not working with reuseForks being True -->
+            <excludes>
+              <exclude>**/TableIdTest.java</exclude>
+              <exclude>**/HadoopCredentialProviderTest.java</exclude>
+              <exclude>**/IdleRatioScanPrioritizerTest.java</exclude>
+              <exclude>**/AssignmentWatcherTest.java</exclude>
+              <exclude>**/TestCfCqSlice.java</exclude>
+              <exclude>**/TestCfCqSliceSeekingFilter.java</exclude>
+            </excludes>
```

Comment:
As long as this list is small, this is a reasonable solution. Another way to exclude them here and include them in the failsafe plugin is to simply rename them to match the failsafe naming pattern, but I like your solution better, because it highlights the problematic ones.

It would be good to do subsequent work to figure out why these have problems with reusing a fork, and to modify the test (or the relevant Accumulo code) to make them play nice with fork reuse.

Reply:
I can create an additional issue go track future work into investigating these tests. That was part of the reason of including them in the pom. Easier to keep track.